app/build.gradle

```diff
@@ -23,6 +23,7 @@ android {
         versionCode 85
         versionName "0.75"
 
+        multiDexEnabled true
```

Comment:
Huh, to many classes?

Reply:
Yes, the many classes come the new Kotlin based Iconics version v4.0. I could just revert to v3.2.5, but sooner or later, like most other projects, we will have to bite the bullet anyway, and I don't think it's that bad anymore.

What do you think?

Reply:
It doesn't really harm anything so I don't care much.